v0/destinations/vero/data/VeroTagsConfig.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "sourceKeys": ["userId", "context.traits.email", "traits.email", "anonymousId"],
```

Comment:
`userId` or `anonymousId` should be mapped here, we should use generic mapping for this

Reply:
Done